runtime/compiler/p/codegen/J9PPCSnippet.cpp

```diff
@@ -2411,8 +2408,6 @@ static uint8_t* initializeCCPreLoadedObjectAlloc(uint8_t *buffer, void **CCPreLo
    TR_ASSERT(cg->getBinaryBufferCursor() - entryLabel->getCodeLocation() == helperSize * PPC_INSTRUCTION_LENGTH,
              "Per-codecache object allocation helper, unexpected size");
 
-   CCPreLoadedCodeTable[TR_ObjAlloc] = entryLabel->getCodeLocation();
```

Comment:
As it stands, all you've removed is the code to put pointers to the snippets in `CCPreLoadedCodeTable`, but the code for the snippets is still being generated and placed in the code cache. You need to remove the `initializeCCPreLoadedObjectAlloc` and `initializeCCPreLoadedArrayAlloc` functions entirely (and the calls to them from `TR::createCCPreLoadedCode`), as they exist only to generate these snippets.